apps/web/pages/api/customer-card.ts

```diff
@@ -0,0 +1,251 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { cardExamples } from "@pages/api/plain/example-cards";
+import crypto from "crypto";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { prisma } from "@calcom/prisma";
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  try {
```

Comment:
Try/catch here is not needed, handlers should get wrapped properly already by the apiHandler